release-notes/SRM/SRM_10.3.0.3.md

```diff
@@ -0,0 +1,20 @@
+---
+uid: SRM_10.3.0.3
+---
+
+# SRM 10.3.0.3
+
+> [!NOTE]
+> This version requires that **DataMiner 10.3.0 [CU0] or higher** is installed. It is not compatible with the DataMiner Feature Release track.
+
+## Enhancements
+
+#### Improved logging in case of invalid entry in Generic DVE Table [ID_38091]
+
+Logging has been improved in case there is an invalid entry in the *Generic DVE Table*. An error will be shown specifying which entry is invalid.
```

Comment:
can you precise this is when using SRM_DiscoverResources script ( used  to provision resources )?